packages/fxa-event-broker/src/app.module.ts

```diff
@@ -28,11 +31,19 @@ import { SentryModule } from './sentry/sentry.module';
       isGlobal: true,
     }),
     FirestoreModule,
-    HealthModule,
+    HealthModule.register({ version }),
```

Comment:
Would it make sense for the health module to do the work of getting the version?

Reply:
It can't, because it doesn't know where its being called from, so it can't search up to find the `package.json` and such.